src/main/java/com/game/Battleship/models/Board.java

```diff
@@ -33,18 +38,20 @@ public void addShip(Ship ship, Integer index){
         if(ship.getHorizontal()) {
             for (int i = 0; i < ship.getSize(); i++) {
                 Cell cell = new Cell();
-                cell.setValue(ship.getType());
+                cell.setValue("Ship");
                 this.boardField.set(index, cell);
                 index++;
             }
         } else {
             for(int i = 0; i < ship.getSize(); i++) {
                 Cell cell = new Cell();
-                cell.setValue(ship.getType());
+                cell.setValue("Ship");
+                cell.setType(ship.getType());
                 this.boardField.set(index, cell);
                 index = index + this.getSize();
             }
         }
+
     }
 
     private List<Cell> generateBoardField(){
```

Comment:
Is the commented code for documentation or is it leftover from development?

Reply:
for documentation